Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Unreleased
 
+# v1.9.0
```

Comment:
This should be in the "Unreleased" section.

Reply:
Done. I was trying to suggest the version number. It is now under "Unreleased (v1.10.0)"